### PR TITLE
MINOR: add internal source topic for tracking

### PR DESCRIPTION
**Pull request by guozhangwang**

### CODE REVIEW

**Comment by guozhangwang** [comment]
@ymatsuda The internal topic names are managed as:

<jobID>-topicName-<RepartitionSuffix>

<jobID>-stateName-<ChangelogSuffix>

File: streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java [hunk 1]
```diff
@@ -460,24 +477,33 @@ private void connectProcessorAndStateStore(String processorName, String stateSto
...
             for (String node : entry.getValue()) {
                 // if the node is a source node, add to the source topics
                 String[] topics = nodeToTopics.get(node);
-                if (topics != null)
+                if (topics != null) {
                     sourceTopics.addAll(Arrays.asList(topics));
 
+                    // if some of the topics are internal, add them to the internal topics
+                    for (String topic : topics) {
+                        if (internalSourceTopics.contains(topic))
```
Comment (ymatsuda) [inline streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java]: You mean `if ( ! internalSourceTopics.contains(topic))`?

Comment (guozhangwang) [inline streams/src/main/java/org/apache/kafka/streams/processor/TopologyBuilder.java]: Oops, wrong name, will fix.

**Comment by ymatsuda** [comment]
lgtm

**Comment by guozhangwang** [comment]
Merged to trunk.

**Pull request closed without merging**